libkbfs/kbfs_ops_test.go

```diff
@@ -75,16 +75,6 @@ func kbfsOpsInit(t *testing.T, changeMd bool) (mockCtrl *gomock.Controller,
 	config.mockBcache = nil
 	config.mockDirtyBcache = nil
 
-	if changeMd {
```

Comment:
Removing this code didn't cause any tests to fail, not even with mock errors. So I think it's obsolete now.